### DISCUSSION TITLE
TumblThree Not picking up a blog

### DISCUSSION THREAD

**Discussion by DimensioX1** [TumblThreeApp/TumblThree#320] [Q&A (Question & Answer)]
I tried to download this blog from Tumblr
https://www.omocat-blog.com/
But the program doesn't let me add it.
Is it because the word Tumblr is not in the address of the blog?
<img width="486" alt="capt" src="https://user-images.githubusercontent.com/17047474/146025752-5c26ac19-2c15-4130-b235-dcf4996c0c22.PNG">

**Comment by thomas694** [comment]
No, TumblThree supports Tumblr custom domains. Actually the hyphen is the problem.
It should be fixed in the next version.

Workaround:
- add a dummy blog, e.g. _cats.tumblr.com_ and close TumblThree
- open file _cats.json_ inside _Index_ folder
- replace "cats.tumblr.com" in _Url_ field with "www.omocat-blog.com"
- replace "cats" with "omocat-blog"
  *  in fields _ChildId_ and _Name_
  * in _Name_ field of file _cats_files.tumblr_
  * in the filenames _cats.tumblr_ and _cats_files.tumblr_
- rename the folder "cats" to "omocat-blog"
- start TumblThree and download

**Comment by DimensioX1** [comment]
Thanks, that worked.

**Comment by thomas694** [comment]
The issue has been fixed and closed. You can still comment. Feel free to ask for reopening the issue if needed.
